src/client/BuilderClient.ts

```diff
@@ -36,15 +39,22 @@ export class BuilderClient {
 
     this.baseUrl = url
 
-    this.fetch = (path: string, init?: RequestInit) => {
+    this.fetch = (
+      path: string,
+      init?: RequestInit & { authenticated?: boolean }
+    ) => {
       const method: string = init?.method ?? path ?? 'get'
       const fullUrl = url + path
+      const authenticated =
+        init?.authenticated !== undefined ? init.authenticated : true
 
       return externalFetch(fullUrl, {
         ...init,
```

Comment:
It will probably not be an issue but the authenticated property is being passed down to the fetch init object.